ci-operator/step-registry/cucushift/installer/rehearse/azure/upi/baselinecaps/v411/provision/cucushift-installer-rehearse-azure-upi-baselinecaps-v411-provision-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-azure-upi-baselinecaps-v411-provision
+  steps:
+  - ref: ipi-install-rbac
+  - chain: ipi-conf-azure
+  - ref: upi-install-azure
+  - ref: enable-qe-catalogsource
+  env:
```

Comment:
missing env name?

Reply:
Do you mean env `ADDITIONAL_CAPABILITY_SET`?